rollup.config.js

```diff
@@ -47,5 +50,13 @@ export default [
       format: 'umd',
       name: 'XStateInterpreter'
     }
+  }),
+  createConfig({
+    input: 'src/index.ts',
+    output: {
+      file: 'dist/xstate.esmodules-browsers.js',
```

Comment:
Can we call this `dist/xstate.esm.js`?

Reply:
Technically we can, but `.esm.js` is often used for "module" field in package.json - which is not the same as this thing here. Didn't want to confuse anyone with this naming.

Reply:
Sorry to nit, I'm just not a fan of the name (not very user-friendly). What about `xstate.web.js`?

Reply:
No problem - I'll change in a moment. I've based my original name on https://babeljs.io/docs/en/babel-preset-env#targetsesmodules